lib/project_types/rails/gem.rb

```diff
@@ -34,18 +34,20 @@ def gem_path(ctx)
       private
 
       def apply_gem_home(ctx)
-        path = File.join(ctx.getenv('HOME'), '.gem', 'ruby', RUBY_VERSION)
+        path = ''
+        out, stat = ctx.capture2e('gem', 'environment', 'home')
+        path = out&.empty? ? '' : out.strip if stat.success?
+        path = File.join(ctx.getenv('HOME'), '.gem', 'ruby', RUBY_VERSION) if path.empty?
```

Comment:
Maybe we could extract `File.join(ctx.getenv('HOME'), '.gem', 'ruby', RUBY_VERSION)` to something like `fallback_to_assumed_ruby_path`?